cdap-app-fabric-tests/pom.xml

```diff
@@ -127,26 +127,12 @@ the License.
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <version>1.7.7</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro-mapred</artifactId>
-      <classifier>hadoop2</classifier>
-      <version>1.7.7</version>
-      <scope>test</scope>
-      <exclusions>
-        <!-- to avoid conflicts with provided netty version from cdap -->
```

Comment:
We no longer need this exclusion?

Reply:
It looks like it's not a dependency anymore:
```
[INFO] \- org.apache.avro:avro-mapred:jar:1.8.2:test
[INFO]    \- org.apache.avro:avro-ipc:jar:1.8.2:test
[INFO]       \- org.apache.velocity:velocity:jar:1.7:test
```

Reply:
@chtyim PTAL